From c6eb4dc5964fdf4086b73b7f0f5dc0d595fb5cf3 Mon Sep 17 00:00:00 2001
From: Dang Vu Duc Hien <dvdh12707@gmail.com>
Date: Tue, 28 Jul 2026 02:52:36 +0700
Subject: [PATCH 07604/15333] staging: rtl8723bs: add blank line after
 declaration

Fix the checkpatch.pl warning:
"WARNING: Missing a blank line after declarations"
in drv_types.h.

Add a blank line between the variable declaration and the subsequent
statement in RTW_ENABLE_FUNC() to comply with the kernel coding style.

Signed-off-by: Dang Vu Duc Hien <dvdh12707@gmail.com>
Link: https://patch.msgid.link/20260727195236.663392-1-dvdh12707@gmail.com
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/staging/rtl8723bs/include/drv_types.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/staging/rtl8723bs/include/drv_types.h b/drivers/staging/rtl8723bs/include/drv_types.h
index b2bbec7769d0..20265770ed77 100644
--- a/drivers/staging/rtl8723bs/include/drv_types.h
+++ b/drivers/staging/rtl8723bs/include/drv_types.h
@@ -389,6 +389,7 @@ struct adapter {
 static inline void RTW_ENABLE_FUNC(struct adapter *padapter, int func_bit)
 {
 	int	df = atomic_read(&adapter_to_dvobj(padapter)->disable_func);
+
 	df &= ~(func_bit);
 	atomic_set(&adapter_to_dvobj(padapter)->disable_func, df);
 }
-- 
2.55.0

